sql/wsrep_mysqld.cc

```diff
@@ -2396,6 +2415,21 @@ int wsrep_to_isolation_begin(THD *thd, const char *db_, const char *table_,
   return ret;
 }
 
+
+bool wsrep_thd_is_in_to_isolation(THD *thd)
+{
+  bool status = FALSE;
+  if (thd)
```

Comment:
I think we can relax this `if (thd)` condition, as this function gets called only when we have a valid thd in the `Global_THD_manager`.

Reply:
Most of the calls that use a thd (like wsrep_thd_is_BF), check the thd ptr.